packages/gafl-webapp-service/src/processors/recurring-pay-helper.js

```diff
@@ -1,5 +1,7 @@
 import { HOW_CONTACTED } from './mapping-constants.js'
 
+const MIN_AGE_FOR_RECURRING_PAYMENT = 18
```

Comment:
As discussed on Slack, this shouldn't be here. Instead, you should use the `isMinor`, `isJunior` and `isSenior` functions in business-rules-lib